client/state/preferences/actions.js

```diff
@@ -28,26 +28,35 @@ export function fetchPreferences() {
 }
```

Comment:
> Surfaced by ESLint, evidenced by my original comment here.

Yes, I am the one that surely will introduce outdated comment :)

> It's not just repeating parameters

Please, explain to me how this is helpful and how more clear it is because of that comment:

```
/**
 * Function used to create an action later dispatched by redux. 
 * @param key - { String | Number } key used as a key in key-value store in redux subtree used to store preferences
 * @param value - { String | Number | Object } value used as a value in key-value store in redux subtree used to store preferences
 * @returns { Object }
 */
export const setPreference = ( key, value ) => ( {
	type: PREFERENCES_SET,
	key,
	value
} );
```

Elaborate comments when they are explaining obvious things, take up cognitive resources and make the code harder to understand, because instead of just seeing familiar pattern, I have to now digest human readable sentence that can be formulated in a million possible ways.